frontend/src/types/api/alerts/def.ts

```diff
@@ -19,7 +19,7 @@ export const defaultSeasonality = 'hourly';
 export interface AlertDef {
 	id?: number;
 	alertType?: string;
-	alert?: string;
+	alert: string;
```

Comment:
intentional?

Reply:
Yes, I believe the alert name should be compulsory. I was getting a type error if we'd keep it optional.